packages/react-components/src/components/VideoGallery/FloatingLocalVideoLayout.tsx

```diff
@@ -218,6 +218,7 @@ export const FloatingLocalVideoLayout = (props: FloatingLocalVideoLayoutProps):
         onChildrenPerPageChange={(n: number) => {
           childrenPerPage.current = n;
         }}
+        parentWidth={parentWidth}
```

Comment:
Does this change also need to be made in the speaker layout?

Reply:
Good catch! Updated.